.gitlab-ci.yml

```diff
@@ -69,6 +69,8 @@ workflow:
 
 check_changelog:
   stage: build
+  variables:
+    GIT_DEPTH: 100
```

Comment:
What if I want 101 commits in my branch? 🙊

Reply:
Where I'm from, we call that a fun opportunity to apply an interactive rebase 😄

(though yes we can absolutely make it longer if we'd prefer)